src/operator/operator_common.h

```diff
@@ -620,6 +620,10 @@ class OpSignature {
       hash = hash * 2 + arr.dtype();
       eles.push_back(arr.dtype());
       AddSign(arr.shape());
+#if DNNL_AARCH64_USE_ACL == 1
+      auto ival = reinterpret_cast<const uint64_t>(arr.storage_handle().dptr);
+      AddSign(ival);
+#endif
```

Comment:
Why is this only needed when `arr.IsMKLDNNData()` is `False`?

Reply:
I checked that this function (arr.IsMKLDNNData()) is called for mkldnn blocked format types https://github.com/apache/incubator-mxnet/blob/7e835a592da902066f2ccb0d74a78adc8e77fdb7/include/mxnet/ndarray.h#L714 and here https://github.com/apache/incubator-mxnet/blob/7e835a592da902066f2ccb0d74a78adc8e77fdb7/src/operator/nn/mkldnn/mkldnn_base.cc#L309 which is not used for ACL builds.

Reply:
@szha, @mseth10, is there anything that needs resolving here?

Reply:
I'm also trying to understand why the hash on pointer address is only needed when building ARM64 with oneDNN.

Reply:
Hi @szha, 
As mentioned we have an issue here #2026, where re-using  primitives causes accuracy failures for an Arm build of MxNET (which is built with Compute Library for Arm). This is a fix we have added to other frameworks like TensorFlow too, issue here https://github.com/tensorflow/tensorflow/issues/47415 and PR raised here https://github.com/tensorflow/tensorflow/commit/06dd30ef987b5511351e979568d4ad06ccdc2844. Please also refer to the discussion with oneDNN team https://github.com/oneapi-src/oneDNN/issues/1113.
As stated previously, this is a workaround we have had to add at the framework level until this issue can be resolved in future releases of Compute Library.

Reply:
OK, thanks for the clarification on the plan to fix. I'm OK with the change and it would be great if you could add a comment in the code on the context and the plan to fix.

Reply:
Done